R/pkg/R/SQLContext.R

```diff
@@ -214,18 +214,45 @@ setMethod("toDF", signature(x = "RDD"),
 #' sc <- sparkR.init()
 #' sqlContext <- sparkRSQL.init(sc)
 #' path <- "path/to/file.json"
-#' df <- jsonFile(sqlContext, path)
+#' df <- read.json(sqlContext, path)
 #' }
 
-jsonFile <- function(sqlContext, path) {
+read.json <- function(sqlContext, ...) {
```

Comment:
I found [```parquetFile```](https://github.com/apache/spark/blob/master/R/pkg/R/SQLContext.R#L270) used var args already, we will get different signature for ```jsonFile``` and ```parquetFile```. Is this as expected? Or we will make break changing for parquetFile? Although I have reverted the code to keep the original signature.